features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/APIProduct.jsx

```diff
@@ -130,6 +130,21 @@ class APIProduct extends Resource {
         });
     }
 
+    /**
+     * Update an api Product via PUT HTTP method, Need to give the updated API Product object as the argument.
+     * @param apiProduct {Object} Updated API Product object(JSON) which needs to be updated
+     */
+    update(apiProduct) {
+        const promised_update = this.client.then((client) => {
```

Comment:
use camel case instead of `promised_update`

Reply:
Fix for `L394` and `L420` as well

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6779/commits/cbbd3e6dfc035b967161efd08bd6bea78f1a5931